examples/eks/README.md

```diff
@@ -0,0 +1,226 @@
+# Connaisseur with EKS
+
+This guide will show you how to set up Connaisseur on your Amazon Elastic Kubernetes Service (EKS) cluster. Unfortunately, the Elastic Container Registry (ECR) does not support Docker Content Trust (DCT) at this point in time. Hence, we will not be using it and for simplicity instead go for Docker Hub, which combines both a registry and a notary server. If you want to setup your own registry and notary, e.g. Harbor, checkout the [Setup Guide](../../setup/README.md), which shows how to work with a (local) instance of Harbor.
+
+## Requirements
+
+We assume you have a running EKS cluster already. Since this tutorial works with the CLI, we furthermore assume you have the `aws`, `docker`, `eksctl`, `git`, `kubectl`, `make` binaries installed and usable, i.e. having run `aws configure`, `aws eks update-kubeconfig` and `docker login`.
+
+> This tutorial was tested on a machine running Ubuntu 20.04 with a EKS cluster based on EC2 instances.
+
+## Starting and testing a deployment of Connaisseur
+
+### 1. Set up environment
+
+First off, we'll export the variables used in this tutorial to allow you to immediately use your own names/URLs instead of relying on our default names. Below, substitute your own values as appropriate:
+
+```bash
+REGISTRY_URL=docker.io
+REPOSITORY_NAME=testingconny
+IMAGE_NAME=testserver
+IMAGE_PATH=$(echo ${REGISTRY_URL}/${REPOSITORY_NAME}/${IMAGE_NAME})
+
+NOTARY_URL=notary.docker.io
+NOTARY_USER=<YOUR-NOTARY-USERNAME/DOCKER-HUB-ID>
+NOTARY_PASSWORD=<YOUR-NOTARY-PASSWORD>
+
+git clone https://github.com/sse-secure-systems/connaisseur.git
+cd connaisseur
+```
+
+### (Optional) 2. Prepare test images
```

Comment:
I see you put this here, since we need the DCT root key later. Technically, it interrupts the flow and would be much better placed at the end before deploying the image. Might enforce creation of a DCT key at this point